### PR TITLE
Fixing #16 with the workaround for VSMac bug https://github.com/mono/…

### PR DESCRIPTION
**Pull request by luizperes**
…monodevelop/pull/5137

### CODE REVIEW

File: FabulousContacts/FabulousContacts.fsproj [hunk 1]
```diff
@@ -25,7 +29,7 @@
     <PackageReference Include="Xam.Plugin.Media" Version="4.0.1.5" />
     <PackageReference Include="Plugin.Permissions" Version="3.0.0.12" />
     <PackageReference Include="Fabulous.XamarinForms.Maps" Version="0.40.3" />
-    <PackageReference Include="FSharp.Core" Version="4.6.2" />
+    <PackageReference Include="FSharp.Core" Version="4.5.2" />
```
Comment (TimLariviere) [inline FabulousContacts/FabulousContacts.fsproj]: Usually I do something like this:
```suggestion
    <PackageReference Include="FSharp.Core" />
    <!-- workaround for VSMac bug https://github.com/mono/monodevelop/pull/5137 -->
    <PackageReference Update="FSharp.Core" Version="4.6.2" />
```

Does it fix the detected downgrade?
(Can't test it for now...)

Comment (luizperes) [reply FabulousContacts/FabulousContacts.fsproj]: The way I did is the way the templates come in VSMac. I tested it, it does work. But if you want, I can use your style, just let me know :)

Comment (TimLariviere) [reply FabulousContacts/FabulousContacts.fsproj]: >But if you want, I can use your style, just let me know :)

Yes, please. Thank you :)

Comment (luizperes) [reply FabulousContacts/FabulousContacts.fsproj]: Done :)

**Pull request merged**